internal/pkg/sessions/cookie_store.go

```diff
@@ -160,14 +165,16 @@ func (s *CookieStore) setSessionCookie(rw http.ResponseWriter, req *http.Request
 
 // LoadSession returns a SessionState from the cookie in the request.
 func (s *CookieStore) LoadSession(req *http.Request) (*SessionState, error) {
+	logger := log.NewLogEntry()
 	c, err := req.Cookie(s.Name)
 	if err != nil {
 		// always http.ErrNoCookie
 		return nil, err
 	}
 	session, err := UnmarshalSession(c.Value, s.CookieCipher)
 	if err != nil {
-		return nil, err
+		logger.WithRequestHost(req.Host).WithError(err)
```

Comment:
I think we need to add a logging action here. IIRC, this doesn't actually log anything but returns a new logger with these default fields, so this might become:

```golang 
logger.WithRequestHost(req.Host).Error(err, "unable to unmarshal session")
```